Header header
uint8 RobotPoseID
uint8 MapPoseID
geometry_msgs/Pose EssentialMatrix 